packages/core/backend/sirius-components-collaborative/src/main/resources/schema/core.graphqls

```diff
@@ -165,6 +165,8 @@ type Mutation {
   deleteObject(input: DeleteObjectInput!): DeleteObjectPayload!
   renameObject(input: RenameObjectInput!): RenameObjectPayload!
   invokeEditingContextAction(input: InvokeEditingContextActionInput!): InvokeEditingContextActionPayload!
+  undo(input : UndoInput!) : UndoPayload!
+  redo(input : RedoInput!) : UndoPayload!
```

Comment:
Everything related to the undo/redo should probably be in a dedicated schema in sirius-web-application since there nothing else in sirius-components